AttitudeManager/Src/AM.cpp

```diff
@@ -1,3 +1,26 @@
 //
 // Created by Anthony Luo on 2022-10-12.
 //
+#include "AM.hpp"
+
+namespace AM {
+
+void AttitudeManager::runControlLoopIteration(
+    AttitudeManagerInput instructions) {
+    // Process Instructions
+
+    // Run Control Algorithms
+    for (auto controller : controller_interfaces) {
+        std::vector<ActuatorOutput> output =
```

Comment:
does this manage to keep track of all the individual outputs for each controls interface so that we can mix it later? It seems like the older values would be overwritten

Reply:
You're correct. I haven't done anything about actually using the data the controller generates.

Reply:
nw, I'll ask @JackGRWD to come up with a solution.

Reply:
This changed in https://github.com/UWARG/ZeroPilot-SW-3/pull/10/commits/9cff2318fa241cde1998c4af1b917c738bf3fc32